examples/fsdp-llm-pretraining/README.md

```diff
@@ -0,0 +1,19 @@
+## Fully Sharded Data Parallel LLM pre-training using Lance text dataset
+
+### Overview
+This is the distributed training version of the LLM-pretraining example. Much of the training code used here was adapted from PyTorch's official [FSDP tutorial](https://pytorch.org/tutorials/intermediate/FSDP_tutorial.html). The key thing to note for distributed training is that you must set the spawn method in order for Lance dataset to work with your training code.
+
+```python
+from multiprocessing import set_start_method
+set_start_method("spawn")
+```
+
+This will ensure that Lance dataset doesn't get stuck when you run your distributed training scripts. More details [here](https://github.com/lancedb/lance/issues/2204).
```

Comment:
We should probably not say "lance dataset doesn't get stuck" because its not a lance problem per se. Maybe better to say "the process doesn't get stuck"